src/js/shield_defs.js

```diff
@@ -1142,6 +1142,8 @@ export function loadShields() {
     notext: true,
   };
 
+  shields["GLCT"] = {};
```

Comment:
There are also LOOP ([`network=GLCT:Loop`](https://taginfo.openstreetmap.org/tags/network=GLCT%3ALoop)) and SCENIC SPUR routes. To implement a loop route, add another entry that passes this definition into a call to `banneredShield()`. The base shield is colored differently, so we’d need separate `overrideByRef`s for these special routes:

[<img src="https://upload.wikimedia.org/wikipedia/commons/e/eb/Lake_Michigan_Circle_Tour_brown.svg" width="100" alt="Lake Michigan Circle Tour Loop">](https://commons.wikimedia.org/wiki/File:Lake_Michigan_Circle_Tour_brown.svg)